packages/core/src/renderer.ts

```diff
@@ -161,12 +161,13 @@ export type ModElementRef<T> =
 export type BaseContext = {
   user?: {
     wallet?: {
-      address?: string;
+      address?: `0x${string}`;
     };
     farcaster?: {
       fid?: string;
     };
   };
+  clientReferralAddress?: `0x${string}`;
```

Comment:
Will add a comment. I'm not too attached to the name, but I do prefer referring to the party implementing Mod as the client since app could be ambiguous